components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -104,6 +106,12 @@ const { nfts, fetchSearch, refetch } = useFetchSearch({
   resetSearch: resetPage,
 })
 
+watch(nfts, () => {
+  listingCartStore.owned = nfts.value
+    .filter((nft) => +(nft?.price ?? 0) === 0)
+    .map((nft) => nftToShoppingCardItem(nft))
```

Comment:
what happening here?

Reply:
Basically assigning all unlisted owned to the store since @exezbcz only want the unlisted ones. This is for the `select all ` function to work.

Reply:
ok

1. it looks like making a computed value that normalizes nft.price to be number would save you the bother of dealing with it again and again

2. 
~~map((nft) => nftToShoppingCardItem(nft))~~
map(nftToShoppingCardItem)

Reply:
Thanks @daiagi `map(nftToShoppingCardItem)` used this instead. 
I had to assign in inside the store since I do all the logic in the store because it is triggered in a different component so computed might not work for this case.

Reply:
I didn't like the watcher here so i refactored it.